### ISSUE TITLE
Opensea refused to connect 

### ISSUE THREAD

**Issue by Craze-dj** [jdhorth/opensea-iframe#3]
It's not working opensea refused to connect

**Comment by jdhorth** [comment]
Yes, OpenSea changed something and now will not display the collections or pages in an iframe. I've created a new branch of this code called 'gem-iframe' and instead have linked it to a Gem.xyz collection of BAYC, instead of OpenSea. Another option to embed from OpenSea is by embedding each NFT individually using the nft card code they provide in the sharing options. Or display from a different marketplace that connects in a iframe. I created a blog post with some instructions on embedding the OpenSea NFTs into a wordpress site which might be of some help. https://www.non-fungi.com/display-nfts-in-wordpress-revisited/ 

I will also be looking into new ways or see if OpenSea has any work arounds for this. I think they were going to launch some kind of profile pages which might be something to look out for.

**Comment by Craze-dj** [comment]
Well I want embed a set of nfts but I don't know how to embed a multiple nfts cards  and arrange them using html.
Well I want to create nft website form my college minor project,if you time help me

**Comment by jdhorth** [comment]
Ok I just pushed a couple of new branches. One has a table with multiple NFT cards displaying, and the other is a HTML page with no table and just the NFT cards, so it's more responsive but they just all stack on top of each other.  If you want to use this code just change the contract address and token ID in the NFT cards to your own NFTs.

**Comment by Craze-dj** [comment]
> Ok I just pushed a couple of new branches. One has a table with multiple NFT cards displaying, and the other is a HTML page with no table and just the NFT cards, so it's more responsive but they just all stack on top of each other.  If you want to use this code just change the contract address and token ID in the NFT cards to your own NFTs.

Can get code for arranging opensea nfts

**Comment by jdhorth** [comment]
This page has the HTML table  4 columns wide: https://github.com/jdhorth/opensea-iframe/blob/nft-cards/index.html

here's example of it live on interweb http://www.non-fun.com/nft/index.html

**Comment by Craze-dj** [comment]
Thanks pal
